model_analyzer/config/generate/model_variant_name_manager.py

```diff
@@ -65,6 +76,29 @@ def get_model_variant_name(self, model_name: str, model_config_dict: Dict,
 
         return (False, model_variant_name)
 
+    def get_ensemble_model_variant_name(
```

Comment:
This looks basically identical to the existing non-exsemble version. Can we combine the two?